interface/forms/dictation/new.php

```diff
@@ -1,29 +1,59 @@
-<!-- Form generated from formsWiz -->
 <?php
-
-
-
+/**
+ *  Dictation Form New
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @author  Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
 
 include_once("../../globals.php");
-include_once("$srcdir/api.inc");
-formHeader("Form: dictation");
-$returnurl = 'encounter_top.php';
+use OpenEMR\Core\Header;
 ?>
-<html><head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<html>
+<head>
+    <title><?php echo xlt('Dictation'); ?></title>
+    <?php Header::setupHeader('ckeditor'); ?>
+    <script src="<?php echo $GLOBALS['webroot'] ?>/library/js/custom/ckeditor_config.js?v=<?php echo $v_js_includes; ?>" type="text/javascript"></script>
+    <?php require $srcdir."/js/xl/ckeditor.js.php" ?>
```

Comment:
If the prior script and this php require (_once?) always go together, may be php can emit the script line.  Otherwise developers are forced to copy a bunch of lines every time.

Also most of guidelines suggest css and js needed by plugins to be included at the end of body.